src/middlewared/middlewared/alert/source/cores.py

```diff
@@ -24,12 +24,6 @@ class CoreFilesArePresentAlertSource(AlertSource):
         "/usr/sbin/smartctl",
     )
     ignore_units = (
-        # Unit: "containerd.service"/"docker.service" is related to k3s.
-        # users are free to run whatever they would like to in containers
-        # and we don't officially support all the apps themselves so we
-        # ignore those core dumps
-        "containerd.service",
-        "docker.service",
```

Comment:
Do we have some other heuristic we're using to determine whether a corefile is generated by an app instead of base system?

Reply:
Containers deployed via containerd have core dumps disabled with their soft/hard limits set to 1 so this is not required anymore.